foxtrot-core/src/main/java/com/flipkart/foxtrot/core/table/impl/TableMapStore.java

```diff
@@ -42,17 +42,18 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.io.Serializable;
 import java.util.Collection;
 import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
-public class TableMapStore implements MapStore<String, Table> {
+public class TableMapStore implements MapStore<String, Table>, Serializable {
     public static final String TABLE_META_INDEX = "table-meta";
     public static final String TABLE_META_TYPE = "table-meta";
     private static final Logger logger = LoggerFactory.getLogger(TableMapStore.class.getSimpleName());
```

Comment:
This is needed because newer version of hazelcast needs object to be serialized so that it can be shared across the cluster. And RestClient inside ElasticsearchConnection doesn't implement Serializable

Reply:
yeah. You're right.